### PR TITLE
(DOCSP-15104): Set up react native redirects for new TOC

### PR DESCRIPTION
**Pull request by mongomoe**
### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-15104

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-15074-Documenting-OpenRealmBehaviorConfiguration/sdk/react-native/

Autobuilder job link:
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?jobId=604a482a75e817aef75b30aa

### Review Guidelines
[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

**Review by cbush — APPROVED** [review]
Looks good at a glance... we'll fix as needed if anything turns up.

File: config/redirects [hunk 1]
```diff
@@ -84,6 +85,37 @@ raw: realm/node/sync-data -> ${base}/sdk/node/examples/sync-changes-between-devi
...
+raw: realm/react-native/migrations -> ${base}/sdk/react-native/examples/modify-an-object-schema/
+raw: realm/react-native/mongodb-realm -> ${base}/sdk/react-native/
+raw: realm/react-native/mongodb -> ${base}/sdk/react-native/examples/query-mongodb/
+raw: realm/react-native/notifications -> ${base}/sdk/react-native/examples/react-to-changes/
+raw: realm/react-native/objects -> ${base}/sdk/react-native/fundamentals/realms/
+raw: realm/react-native/open-a-realm -> ${base}/sdk/react-native/examples/open-and-close-a-local-realm/
+raw: realm/react-native/query-engine -> ${base}/sdk/react-native/advanced/query-engine/
+raw: realm/react-native/quick-start -> ${base}/sdk/react-native/quick-start/
+raw: realm/react-native/reads -> ${base}/sdk/react-native/examples/read-and-write-data/
+raw: realm/react-native/realms -> ${base}/sdk/react-native/
```
Comment (nathan-contino-mongo) [inline config/redirects:112]: Should this redirect to ` ${base}/sdk/react-native/fundamentals/realms/`?

File: config/redirects [hunk 2]
```diff
@@ -84,6 +85,37 @@ raw: realm/node/sync-data -> ${base}/sdk/node/examples/sync-changes-between-devi
...
+raw: realm/react-native/init-realmclient -> ${base}/sdk/react-native/examples/connect-to-mongodb-realm-backend-app/
+raw: realm/react-native/install -> ${base}/sdk/react-native/install/
+raw: realm/react-native/link-identities -> ${base}/sdk/react-native/advanced/link-identities/
+raw: realm/react-native/manage-email-password-users -> ${base}/sdk/react-native/examples/manage-email-password-users/
+raw: realm/react-native/migrate -> ${base}/sdk/react-native/fundamentals/schema-versions-and-migrations/
+raw: realm/react-native/migrations -> ${base}/sdk/react-native/examples/modify-an-object-schema/
+raw: realm/react-native/mongodb-realm -> ${base}/sdk/react-native/
+raw: realm/react-native/mongodb -> ${base}/sdk/react-native/examples/query-mongodb/
+raw: realm/react-native/notifications -> ${base}/sdk/react-native/examples/react-to-changes/
+raw: realm/react-native/objects -> ${base}/sdk/react-native/fundamentals/realms/
```
Comment (nathan-contino-mongo) [inline config/redirects:107]: Should this instead redirect to objects/schemas?

**Review by nathan-contino-mongo — APPROVED** [review]
LGTM with two small nits/questions

File: config/redirects (hunk 1 above)
Comment (mongomoe) [reply config/redirects:112]: Good catch, it should be 👍  Also fixed the adjacent redirect for node.

File: config/redirects (hunk 2 above)
Comment (mongomoe) [reply config/redirects:107]: It should not be, the "Object Types & Schemas" content that was formally on its own page is now a subsection.

**Pull request merged**